composer.json

```diff
@@ -3,14 +3,20 @@
     "license": "MIT",
     "type": "sulu-bundle",
     "description": "Bundle that provides controllers and services for using Sulu as headless content management system",
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "git@github.com:alexander-schranz/sulu.git"
+        }
+    ],
     "require": {
         "php": "^7.3 || ^8.0",
-        "sulu/sulu": "^2.4 || ^2.5@dev",
-        "symfony/config": "^4.4 || ^5.0 || ^6.0",
-        "symfony/dependency-injection": "^4.4 || ^5.0 || ^6.0",
-        "symfony/framework-bundle": "^4.4 || ^5.0 || ^6.0",
-        "symfony/http-foundation": "^4.4 || ^5.0 || ^6.0",
-        "symfony/http-kernel": "^4.4 || ^5.0 || ^6.0"
+        "sulu/sulu": "dev-feature/symfony-7-support as 2.6.3",
```

Comment:
This will currently test the Symfony 7 compatibility as seen in previous commit the lower symfony versions run. This change need to be reverted when sulu/sulu#7156 is merged.